### ISSUE TITLE
Does gRPC support Digest Authentication ?

### ISSUE THREAD

**Issue by shubhrojroy** [grpc/grpc-java#3842]
I am trying to figure out if the BigTable HBase client supports digest authentication. The google bigtable team asked me to check here.

### What version of gRPC are you using?
Whatever the latest bigtable hbase client is using.

### What did you expect to see?
digest authentication.

**Comment by sduskis** [comment]
Cloud Bigtable currently uses grpc 1.7.  Is there any way for a proxy that uses 1.1 digest authentication to work well with a grpc/netty client?

**Comment by ejona86** [comment]
No. Currently there is no support for Digest authentication for proxies. Digest authentication requires going through the full authentication handshake, so isn't trivially supported.

**Comment by ejona86** [comment]
Should this be taken as a feature request to add digest support to the proxy code?

**Comment by shubhrojroy** [comment]
That would be really helpful. Currently our proxy can only interact via digest authentication. So in this case we would actually end up whitelisting IPs and opening ports instead of using a proxy.
